x/epochs/keeper/keeper_test.go

```diff
@@ -1,24 +1,62 @@
 package keeper_test
 
 import (
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	epochskeeper "github.com/osmosis-labs/osmosis/x/epochs/v14/keeper"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/suite"
 
-	"github.com/osmosis-labs/osmosis/v14/app/apptesting"
-	"github.com/osmosis-labs/osmosis/v14/x/epochs/types"
+	"github.com/osmosis-labs/osmosis/x/epochs/v14/types"
 )
 
 type KeeperTestSuite struct {
-	apptesting.KeeperTestHelper
-	queryClient types.QueryClient
+	suite.Suite
+	Ctx          sdk.Context
+	EpochsKeeper *epochskeeper.Keeper
+	queryClient  types.QueryClient
 }
 
 func (suite *KeeperTestSuite) SetupTest() {
-	suite.Setup()
-	suite.queryClient = types.NewQueryClient(suite.QueryHelper)
+	ctx, epochsKeeper := Setup()
+	suite.Ctx = ctx
+	suite.EpochsKeeper = epochsKeeper
+	queryRouter := baseapp.NewGRPCQueryRouter()
+	cfg := module.NewConfigurator(nil, nil, queryRouter)
+	types.RegisterQueryServer(cfg.QueryServer(), epochskeeper.NewQuerier(*suite.EpochsKeeper))
+	suite.queryClient = types.NewQueryClient(&baseapp.QueryServiceTestHelper{
+		GRPCQueryRouter: queryRouter,
+		Ctx:             suite.Ctx,
+	})
```

Comment:
If these are changes due to the epoch module now being a separate module, would it be possible to edit `apptesting.KeeperTestHelper` so that it now uses the updated epochs package?

Reply:
yes, the problem of having the `apptesting` or any osmosis/app which imports epochs again is that it gives some errors like https://github.com/osmosis-labs/osmosis/pull/2544#discussion_r959544446
And these apptesting.KeeperTestHelper is already importing the epochs package... its all changed to osmosis-labs/osmosis/x/epochs/v14 for now..

Reply:
yeah, can't really make x/epochs use apptesting.

We could at most introduce a dependency into `osmoutils`, and that could do it. 

But I don't feel personally too worried about this, I think it'll get better with some upcoming SDK changes.